server/processors/src/main/java/org/hawkular/apm/server/processor/nodedetails/NodeDetailsDeriver.java

```diff
@@ -164,4 +158,44 @@ protected void deriveNodeDetails(Trace trace, List<Node> nodes, List<NodeDetails
         commonProperties.addAll(trace.getProperties(Constants.PROP_PRINCIPAL));
         return commonProperties;
     }
+
+    /**
+     * This method calculates the elapsed time associated with the child nodes of the
+     * supplied parent node. If the supplied node is not a parent, or if the child nodes are
```

Comment:
I'll update the comment - basically just means that if the supplied node has no child nodes, then it elapsed time  will equal actual time.